commands_to_load/help.json

```diff
@@ -1,46 +1,46 @@
-{ "commands" : [

```

Comment:
you messed up with help.json
please do 
`git fetch`
`git checkout origin/master -- commands_to_load/help.json`

after doing that, add the command you want to add, but there has been a convention change in the file. if you may notice, the value for `access` has been changed.

These now map to either `role` or `channel.` What this basically means is if the access to that command is based on the user's `role`, then the value is `role` the there is an additional key-value pair like so `( "role": "<what _role>")`
There is also another possibility. If the access is based on the user's permission in the `channel` that they called the command, then the value is `channel` for key `access` and there is an additional key-value pair like so `("channel":"<what_permission_in_channel_is_needed_for_access")`

Reply:
done